wp-content/themes/borderzine/inc/widgets/class-borderzine-6-col-widget.php

```diff
@@ -0,0 +1,281 @@
+		$preserve = $post;
+
+		$posts_term = of_get_option( 'posts_term_plural', 'Posts' );
+
+		// Add the link to the title.
+		$title = apply_filters( 'widget_title', empty( $instance['title'] ) ? '' : $instance['title'], $instance, $this->id_base );
+
+		echo $args['before_widget'];
+
+        if ( $title && ! empty( $title ) ) echo $args['before_title'] . $title . $args['after_title'];
+        
+        if( ! empty( $instance['linkurl'] ) && ! empty( $instance['linktext'] ) ) {
+			echo '<p class="morelink btn btn-primary"><a href="' . esc_url( $instance['linkurl'] ) . '">' . esc_html( $instance['linktext'] ) . '</a></p>';
+		}
+
+		$thumb = 'rect_thumb_half';
+		$excerpt = isset( $instance['excerpt_display'] ) ? $instance['excerpt_display'] : 'num_sentences';
+
+		$query_args = array (
+			'post__not_in' 	 => get_option( 'sticky_posts' ),
+			'posts_per_page' => isset( $instance['num_posts'] ) ? $instance['num_posts'] : 6,
+			'post_status'	=> 'publish'
+		);
+
+		if ( isset( $instance['avoid_duplicates'] ) && $instance['avoid_duplicates'] === 1 ) {
+			$query_args['post__not_in'] = $shown_ids;
+		}
+		if ( ! empty( $instance['cat'] ) ) {
+			$query_args['cat'] = $instance['cat'];
+		}
+		if ( ! empty( $instance['tag'] ) ) {
+			$query_args['tag'] = $instance['tag'];
+		}
+		if ( ! empty( $instance['author'] ) ) {
+			$query_args['author'] = $instance['author'];
+		}
+		if ( ! empty( $instance['taxonomy'] ) && ! empty( $instance['term'] ) ) {
+			$query_args['tax_query'] = array(
+				array(
+					'taxonomy'	=> $instance['taxonomy'],
+					'field' 	=> 'slug',
+					'terms' 	=> $instance['term']
+				)
+			);
+		}
+
+		echo '<ul>';
+
+		$my_query = new WP_Query( $query_args );
+
+		if ( $my_query->have_posts() ) {
+
+			$output = '';
+
+			while ( $my_query->have_posts() ) : $my_query->the_post(); $shown_ids[] = get_the_ID();
+
+				// wrap the items in li's.
+				$output .= '<li>';
+
+				$context = array(
+					'instance' => $instance,
+					'thumb' => $thumb,
+					'excerpt' => $excerpt
+				);
+
+				ob_start();
+				largo_render_template( 'partials/widget', 'content', $context );
+				$output .= ob_get_clean();
+
+				// close the item
+				$output .= '</li>';
+
+			endwhile;
+
+			// print all of the items
+			echo $output;
+
+		} else {
+			printf( __( '<p class="error"><strong>You don\'t have any recent %s.</strong></p>', 'largo' ), strtolower( $posts_term ) );
+		} // end more featured posts
+
+		// close the ul
+		echo '</ul>';
+
+		echo $args['after_widget'];
+
+		// Restore global $post
+		wp_reset_postdata();
+		$post = $preserve;
+	}
+
+	function update( $new_instance, $old_instance ) {
+		$instance = $old_instance;
+		$instance['title'] = sanitize_text_field( $new_instance['title'] );
+		$instance['num_posts'] = intval( $new_instance['num_posts'] );
+		$instance['avoid_duplicates'] = ! empty( $new_instance['avoid_duplicates'] ) ? 1 : 0;
+		$instance['thumbnail_display'] = sanitize_key( $new_instance['thumbnail_display'] );
```

Comment:
> Undefined index: thumbnail_display in /wp-content/themes/borderzine/inc/widgets/class-borderzine-6-col-widget.php 

https://github.com/INN/umbrella-borderzine/commit/f766833d9593870549f624102f4c410abd663258 missed a few references to the removed variable; this is what's preventing the widget from saving in https://github.com/INN/umbrella-borderzine/pull/75#issuecomment-556935358

Reply:
https://github.com/INN/umbrella-borderzine/pull/75/commits/14e976bb9573328411e64196b5562cce1137ff9b should solve that